docs/api-and-integration/octopus.migrator.exe-command-line/import.md

```diff
@@ -10,24 +10,6 @@ The export must have been made from an Octopus Server running the same release v
 Usage:
 
 ```text
-Octopus.Migrator import [<options>]
```

Comment:
It looks like this is broken again :(

Reply:
Yeah, still borked. Just trying to get https://github.com/OctopusDeploy/OctopusDeploy/pull/3684 in.

Reply:
Sorry - meant to add a comment on this PR.